evansmusomi/react-native-102
# Manages Input State and Validates Auth Screen Controls

*What this does*
- Manages input state on Auth screen
- Validates user input on Auth screen

*How to verify*
- The submit button should be disabled by default when you load the Auth Screen
- Typing an invalid entry in an input field should make the input field red
- Typing a valid entry in an input field should make the input field white (normal)
- When all fields are correctly filled in, the submit button should be enabled
